File: libgdx_ninja_runner/core/src/com/nopalsoft/ninjarunner/screens/SettingsScreen.java
package com.nopalsoft.ninjarunner.screens;

import com.badlogic.gdx.Game;
import com.badlogic.gdx.Input.Keys;
import com.nopalsoft.ninjarunner.game.GameScreen;

public class SettingsScreen extends Screens {

    public SettingsScreen(Game game) {
        super(game);
    }

    @Override
    public void draw(float delta) {
    }

    @Override
    public void update(float delta) {
    }

    @Override
    public boolean keyDown(int keycode) {
        if (keycode == Keys.ESCAPE || keycode == Keys.BACK) {
            changeScreenWithFadeOut(GameScreen.class, game);
            return true;
        }
        return super.keyUp(keycode);
    }

    @Override
    public void hide() {
    }
}

File: Dont_Tap/core/src/com/nopalsoft/ninjarunner/screens/SettingsScreen.kt
package com.nopalsoft.ninjarunner.screens

import com.badlogic.gdx.Game
import com.badlogic.gdx.Input
import com.nopalsoft.ninjarunner.game.GameScreen

class SettingsScreen(game: Game) : Screens(game) {
    override fun draw(delta: Float) {
    }

    override fun update(delta: Float) {
    }

    override fun keyDown(keycode: Int): Boolean {
        if (keycode == Input.Keys.ESCAPE || keycode == Input.Keys.BACK) {
            changeScreenWithFadeOut(GameScreen::class.java, game)
            return true
        }
        return super.keyUp(keycode)
    }

    override fun hide() {
    }
}